### PR TITLE
update user_guide

### LINKED ISSUE #621: inception server not working
i follow the step of `Serve a model using TensorFlow Serving` in [user_guide](https://github.com/kubeflow/kubeflow/blob/master/user_guide.md)

`kubectl get svc -n kubeflow inception`

```
NAME        TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)             AGE
inception   ClusterIP   10.97.23.123   <none>        9000/TCP,8000/TCP   13m
```

`kubectl get po -n kubeflow`

```
NAME                                READY     STATUS    RESTARTS   AGE
ambassador-849fb9c8c5-288fp         2/2       Running   0          3d
ambassador-849fb9c8c5-6ltq5         2/2       Running   0          3d
ambassador-849fb9c8c5-hzc4k         2/2       Running   0          3d
inception-7bc4df4546-6b4gj          1/1       Running   0          14m
jupyter-ciscoai                     1/1       Running   0          3d
```

it seems running!

also i did `kubectl describe pod/inception-7bc4df4546-6b4gj -n kubeflow`, and it shows:

```
Events:
  Type    Reason                 Age   From               Message
  ----    ------                 ----  ----               -------
  Normal  Scheduled              55s   default-scheduler  Successfully assigned inception-7bc4df4546-6b4gj to ciscoai
  Normal  SuccessfulMountVolume  55s   kubelet, ciscoai   MountVolume.SetUp succeeded for volume "default-token-qwmpv"
  Normal  Pulled                 55s   kubelet, ciscoai   Container image "gcr.io/kubeflow-images-staging/tf-model-server-cpu:v20180327-995786ec" already present on machine
  Normal  Created                55s   kubelet, ciscoai   Created container
  Normal  Started                54s   kubelet, ciscoai   Started container
```

then i use `kubectl get pod/inception-7bc4df4546-6b4gj -n kubeflow -o yaml` to get portIP is `192.168.175.202`(it succeed for `jupyter`), so i think `192.168.175.202:9000` is correct.

then i deployed inception-client, after that: `python label.py -s 192.168.175.202 -p 9000 images/sleeping-pepper.jpg`, but the result is not nice

```
Traceback (most recent call last):
  File "label.py", line 82, in <module>
    main(args.images, args.server, args.port)
  File "label.py", line 56, in main
    result = stub.Predict(request, 10.0)  # 10 secs timeout
  File "/home/ciscoai/kubeflow/components/k8s-model-server/inception-client/client/local/lib/python2.7/site-packages/grpc/beta/_client_adaptations.py", line 310, in __call__
    self._request_serializer, self._response_deserializer)
  File "/home/ciscoai/kubeflow/components/k8s-model-server/inception-client/client/local/lib/python2.7/site-packages/grpc/beta/_client_adaptations.py", line 196, in _blocking_unary_unary
    raise _abortion_error(rpc_error_call)
grpc.framework.interfaces.face.face.AbortionError: AbortionError(code=StatusCode.UNAVAILABLE, details="Connect Failed")
```

anyone can help me?

--

`ks version`

```
ksonnet version: 0.9.2
jsonnet version: v0.9.5
client-go version: 1.8
```

--

`kubectl version`

```
Client Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.0", GitCommit:"fc32d2f3698e36b93322a3465f63a14e9f0eaead", GitTreeState:"clean", BuildDate:"2018-03-26T16:55:54Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.0", GitCommit:"fc32d2f3698e36b93322a3465f63a14e9f0eaead", GitTreeState:"clean", BuildDate:"2018-03-26T16:44:10Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}
```

### PR DESCRIPTION
**Pull request by lluunn**
The inception model on GCS should be public, but some user might run into problem reading it.

Fix #621 
/cc @jlewi

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/646)

### CODE REVIEW

**Comment by pdmack** [comment]
Do we have a good link for the minimum auth setup required?
[This](https://cloud.google.com/storage/docs/authentication)?

File: user_guide.md [hunk 1]
```diff
@@ -322,6 +322,10 @@ inception   LoadBalancer   10.35.255.136   ww.xx.yy.zz   9000:30936/TCP   28m
 
 In this example, you should be able to use the inception_client to hit ww.xx.yy.zz:9000
 
+The model at gs://kubeflow-models/inception is public accessible. However, if your environment doesn't
```
Comment (jlewi) [inline user_guide.md]: publicly

File: user_guide.md [hunk 2]
```diff
@@ -322,6 +322,10 @@ inception   LoadBalancer   10.35.255.136   ww.xx.yy.zz   9000:30936/TCP   28m
...
+have google authentication setup, TF serving will have problem reading the model.
```
Comment (jlewi) [inline user_guide.md]: Do we have a link to what it means to have GCP credentials?
"TF serving will not be able to read the model"

Comment (pdmack) [reply user_guide.md]: nit: same as above "publicly"

**Comment by lluunn** [comment]
Added more explanation and a link. PTAL, thanks

**Pull request merged**